yt_dlp/extractor/lbry.py

```diff
@@ -67,11 +67,12 @@ def _parse_stream(self, stream, url):
             'release_timestamp': ('value', 'release_time', {int_or_none}),
             'tags': ('value', 'tags', ..., {lambda x: x or None}),
             'duration': ('value', stream_type, 'duration', {int_or_none}),
-            'channel': ('signing_channel', 'value', 'title', {str}),
             'channel_id': ('signing_channel', 'claim_id', {str}),
         })
 
-        channel_name = traverse_obj(stream, ('signing_channel', 'name', {str}))
+        info['uploader_id'] = traverse_obj(stream, ('signing_channel', 'normalized_name', {str}))
+        channel_name = traverse_obj(stream, ('signing_channel', (('value', 'title'), 'name'), {str}), get_all=False)
+        info['channel'] = channel_name
         if channel_name and info.get('channel_id'):
             info['channel_url'] = self._permanent_url(url, channel_name, info['channel_id'])
```

Comment:
`channel_name` should not be changed to `stream['signing_channel']['value']['title']`, since it is a completely different value than what it was before, and since `channel_name` is used to construct the `channel_url`, which would then be incorrect.

We can simplify all of this by using the `uploader_id` value already extracted by the other suggestion
```suggestion
        if info.get('uploader_id') and info.get('channel_id'):
            info['channel_url'] = self._permanent_url(url, info['uploader_id'], info['channel_id'])
```